js/app/src/Blocks.svelte

```diff
@@ -304,6 +313,21 @@
 					}
 
 					if (status.stage === "error") {
+						if (status.message) {
+							console.log();
+							messages = [
+								{
+									type: "error",
+									message: status.message.substring(
```

Comment:
This is causing the first and last characters of the error message (not quotes) to be stripped if the user doesn't raise a `gr.Error`. I made the following modifications to your demo (load event error + show_error=True):

```python
def load_error():
    raise ValueError("This is an error!")

with gr.Blocks() as demo:
    input = gr.TextArea()
    output = gr.TextArea()
    gr.Button().click(fn=one, inputs=input, outputs=output).then(two).then(three).then(
        four
    ).then(five).then(six).then(seven)
    demo.load(load_error, None, None)

demo.launch(show_error=True)
````

<img width="1434" alt="Screenshot 2023-06-09 at 1 36 22 PM" src="https://github.com/gradio-app/gradio/assets/41651716/a1584c10-60ea-47d4-a93c-4fcdfb58b0a1">

Other than that this LGTM @pngwn !

Reply:
I've adjusted this to use regex-based replacement. It will now only trim if the message begins and ends with a single quote.